src/App.js

```diff
@@ -1,85 +1,230 @@
 import React from 'react';
 
-function App() {
-  return (
-    <section className="todoapp">
-      <header className="header">
-        <h1>todos</h1>
-
-        <input
-          className="new-todo"
-          placeholder="What needs to be done?"
+import Header from './components/Header/Header';
+import AddForm from './components/AddForm/AddForm';
+import ListTodos from './components/ListTodos/ListTodos';
+import Footer from './components/Footer/Footer';
+
+class App extends React.Component {
+  state = {
+    todos: [],
+    activeTodo: [],
+    completedTodo: [],
+    activeList: 'all',
+    currentId: 1,
+    selectAll: false,
+  }
+
+  componentDidMount() {
+    this.valueOfLocalStorage();
+    localStorage.setItem(`${0}`, JSON.stringify({ startItem: 'startItem' }));
+  }
+
+  componentDidUpdate() {
+    localStorage.clear();
+    this.state.todos.forEach((todo, i) => (
+      localStorage.setItem(`${i + 1}`, JSON.stringify({ ...todo }))
+    ));
+  }
+
+  validationInput = (newTodoValue) => {
```

Comment:
validation is a noun, verbs are used for function names, use the name I suggested you before